docs/spec/spec.md

```diff
@@ -422,21 +422,21 @@ public function main() returns error? {
 }
 ```
 
-# 5. gRPC Security
+# 5. gRPC security
 
-## 5.1 Authentication and Authorization
+## 5.1 Authentication and authorization
 
 There are two ways to enable authentication and authorization in gRPC.
 1. Declarative approach
 2. Imperative approach
 
-### 5.1.1 Declarative Approach
+### 5.1.1 Declarative approach
 
 This is also known as the configuration-driven approach, which is used for simple use cases, where users have to provide a set of configurations and do not need to be worried more about how authentication and authorization works. The user does not have full control over the configuration-driven approach.
 
 The service configurations are used to define the authentication and authorization configurations. Users can configure the configurations needed for different authentication schemes and configurations needed for authorizations of each authentication scheme. Also, the configurations can be provided at the service level. The priority will be given from bottom to top. Then, the auth handler creation and request authentication/authorization is handled internally without user intervention. The requests that succeeded both authentication and/or authorization phases according to the configurations will be passed to the business logic layer.
 
-#### 5.1.1.1 Service - Basic Auth - File User Store
+#### 5.1.1.1 Service - basic auth - file user store
```

Comment:
```suggestion
#### 5.1.1.1 Service - Basic Auth - file user store
```

Reply:
This is the correct term https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#security_of_basic_authentication